docs/dev/libvirt-howto.md

```diff
@@ -312,6 +312,18 @@ Depending on your libvirt version you might encounter [a race condition][bugzill
 ```
 This is also being [tracked on the libvirt-terraform-provider][tfprovider_libvirt_race] but is likely not fixable on the client side, which is why you should upgrade libvirt to >=4.5 or a patched version, depending on your environment.
 
+### `Error creating libvirt volume for Ignition worker.ign`
```

Comment:
This might be a bug in libvirt destroy. Can you open issue with debug logs from cluster-destroy for a cluster with workers.